fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if(process.platform !== 'darwin') {
+  var err = new Error("Cannot find module 'fsevents'")
```

Comment:
Currently, on non darwin systems, the fsevents module is not physically installed. When a dependent `require()s` it, the get the above exception from node's module loader.

The intent here is to simulate that same exception as close as possible, to increase odds of compatibility; for example if a dependent is actually looking at the message and/or err code for some reason.

Where do you want single quotes used specifically? Used doubles in error message because it has singles in it.